src/Bicep.Core/Parsing/Parser.cs

```diff
@@ -609,26 +609,93 @@ private SyntaxBase PrimaryExpression(ExpressionFlags expressionFlags)
             }
         }
 
+        private SyntaxBase ParenthesizedExpressionOrLambda(ExpressionFlags expressionFlags)
```

Comment:
>ParenthesizedExpressionOrLambda [](http://example.com/codeflow?start=27&length=31)

Should also update our pseudo grammar file

Reply:
Great point